core/scripts/vrfv2/testnet/main.go

```diff
@@ -28,6 +28,17 @@ func panicErr(err error) {
 	}
 }
 
+func parseArgs(fset *flag.FlagSet, args []string, requiredArgs []string) {
```

Comment:
nit: Could make this "...string" instead of []string, then callers would do something like:

parseArgs(coordinatorDeployCmd, os.Args[2:], "link-address", "bhs-address", "link-eth-feed")

which looks a bit cleaner IMO

Reply:
Also I wonder if parseArgs should be in a shared spot, definitely seems useful for any future scripts too.